frontend/app/views/spree/home/index.html.erb

```diff
@@ -1,14 +1,139 @@
-<% @body_id = 'home-page' %>
-<% content_for :sidebar do %>
-  <div data-hook="homepage_sidebar_navigation">
-    <%= render partial: 'spree/shared/taxonomies' %>
+<div class="homepage-hero-image mb-3 position-relative">
+  <img class="lazyload hero-image position-absolute"
+    data-src="<%= asset_path('homepage/summer_collection.jpg') %>"
```

Comment:
are we going to add those assets as well? I think there's going to be an error in case of missing images.

Reply:
you're right, the images were introduced in https://github.com/spree/spree/pull/9649